scipy/optimize/_minimize.py

```diff
@@ -599,51 +599,78 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
 
     if bounds is not None:
         bounds = standardize_bounds(bounds, x0, meth)
+        if isinstance(bounds, Bounds):
+            lb = bounds.lb
+            ub = bounds.ub
+        else:
+            bounds_a = np.array(bounds)
+            lb = bounds_a[:, 0]
+            ub = bounds_a[:, 1]
+
+        i_fixed = np.abs(ub - lb) < 1e-8
```

Comment:
```suggestion
        i_fixed = np.abs(ub - lb) < tol
```
Some sort of tolerance